bin/cron/deliver_poste_messages

```diff
@@ -8,8 +8,8 @@ require 'nokogiri'
 require src_dir 'forms'
 require src_dir 'abort_email_error'
 
-BATCH_SIZE = 6000
-MAX_THREAD_COUNT = 30
+BATCH_SIZE = 500000
```

Comment:
nit: Use `_` to separate, e.g., `500_000`.

Given that these seem to have worked, I'll call this tested (in production)?